lib/datadog/tracing/contrib/rack/middlewares.rb

```diff
@@ -23,6 +23,8 @@ module Rack
         # in the Rack environment so that it can be retrieved by the underlying
         # application. If request tags are not set by the app, they will be set using
         # information available at the Rack level.
+        #
+        # rubocop:disable Metrics/ClassLength
```

Comment:
I can get the argument for method length (with exceptions), but I'm wondering if there's really such a thing as a class with "too much lines".

Reply:
I don't think I've ever decided to break down a class when `Metrics/ClassLength` is triggered, only ever added the cop exclusion tag.

If you want to remove the cop, I'd ✅ it.

Reply:
Seems like there are not too many of them, so it can reasonably fit into this PR.